documentation/crds_users_guide/source/library_use.rst

```diff
@@ -106,7 +106,7 @@ reference file paths:
                       
           context :   The name of the pipeline context mapping which should be
                   used to define best reference lookup rules,  or None.  If 
-                  `context` is None,  use the latest operational pipeline mapping.
+                  `context` is None,  use the latest pipeline mapping.
```

Comment:
"If `context` is None, use the latest pipeline mapping (for JWST, this is the build context mapping).